e2e/cypress/integration/messaging/input_box_exapands_50_per_spec.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Input box expands upto 50 % only', () => {
+    before(() => {
+        // # Login and navigate to town-square
+        cy.toMainChannelView('user-1');
+    });
+
+    it(' M18709 - Input box expands upto 50 % of total screen size', () => {
+        // # get viewHeiht to compare
+        const viewHeight = Cypress.config('viewportHeight');
+
+        // # type  \n until the textbox reachs to max height.
```

Comment:
Before this line, it should post a message first and then click "reply" to open the thread

Reply:
done